nginx-fem-redirects.conf

```diff
@@ -1,4 +1,23 @@
 set $fe_project_uri "https://fe-project.zooniverse.org";
+
+location ~* ^/(?:projects|about)/_next/.+?$ {
```

Comment:
I think these should be split out as they need to proxy to different targets. My understanding is 
- `/projects/_next` proxies to `fe-project.zooniverse.org`
- `/about/_next` proxies to `fe-content-pages.zooniverse.org`

I may be misunderstanding the paths here

Reply:
D'oh! You're absolutely right. I thought I was being economical by combining rules here.